### ISSUE TITLE
when logged in, users see a different page

### ISSUE THREAD

**Issue by karissa** [datproject/datproject.org#414]
Once you've logged in, datproject.org should show a different landing page.

Let's say it is `/list` for now, which is a list of all of the dats. How does that sound?

note: this is not a redirect

**Comment by karissa** [comment]
this depends upon #404

**Comment by karissa** [comment]
This is low priority not sure if we need it anymore because we switched the header to be persistent
